yieldcurity_standard.md

```diff
@@ -0,0 +1,207 @@
+# The Yieldcurity Standard 
+
+Opinionated **security**, **code quality** standards, and **code review policies and procedures** for **Solidity** smart contracts.  Inspired by and fully incorporating [The Solcurity Standard](https://github.com/Rari-Capital/solcurity).
```

Comment:
is this 100% copied?  I think maybe a fork is better.  then make make yield/solcurity  fork a dependency.  rules that we change can use a modified code maybe?  `V9` => `V9Yield` or `V9-Y`

Reply:
I copied Solcurity then added my own material from line 170 - 203 covering:
- Testing,
-  Other points of consideration, 
- Deployment, 
- Post-Deployment

Reply:
Oh, and i changed the index for the Contract section to K1,K2...